src/Slices/ServiceInstanceDetails/UI/Components/AttributesComponents/NoDataState.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { words } from "@/UI";
+import { ErrorView } from "@/UI/Components";
+
+/**
+ * React Component containing the errorView that is shared
+ * amongst the different views of the attributesTab
+ */
+export const NoDataState: React.JSX.Element = (
+  <ErrorView message={words("instanceDetails.page.noData")} />
```

Comment:
If I understood the Provider right, there is no other ErrorView for logsQuery, if that's correct, then maybe we could take eventual errorMesage from the context to display it here, and that general message as a fallback

Reply:
I planned to make this more generic when implementing the next tabs, and have something more aligned